Cargo.toml

```diff
@@ -18,8 +18,11 @@ regex = "1.3.6"
 lazy_static = "1.4.0"
 humantime = "2.0.0"
 log = "0.4.8"
+actix-web = "2.0"
```

Comment:
It feels wrong for the common library to rely on actix. You'd be gratuitously increasing compile times for everything downstream, which might even be i.e. just a small scene for the graphical client. I would hide anything that requires knowledge of actix-web behind a cargo feature.

Reply:
Yeah I understand this. I'll gate it behind a cargo feature